
syntax = "proto3";

message Number {
    float value = 1;
}

service Car {
    rpc GetCar(Number) returns (Number) {}
}